pages/linux/swaybg.md

```diff
@@ -0,0 +1,16 @@
+# swaybg
+
+> Wallpaper tool for Wayland compositors.
+> More information: <https://github.com/swaywm/swaybg>.
```

Comment:
The README doesn't contain a lot of info, I think the man page is better. It's even referenced in the README:
> See the man page, swaybg(1), for instructions on using swaybg.

```suggestion
> More information: <https://github.com/swaywm/swaybg/blob/master/swaybg.1.scd>.
```